tin/templates/assignments/manage_students.html

```diff
@@ -0,0 +1,27 @@
+{% extends "base.html" %}
+{% load static %}
+
+{% block title %}
+  Turn-In: Manage Students
+{% endblock %}
+
+
+{% block main %}
+  <h2>Manage Students</h2>
+
+  <div class="student-list">
+    <ul class="linebreak-list linebreak-last">
+      {% for student in students %}
+        <li>
+          <div class="left">
+          {{ student.last_name }}, {{ student.first_name }}
```

Comment:
Might want to include Ion usernames (matching the standard name format) and links to view individual students' submission history for the assignment (`student_submissions`), maybe also their most recent submission and score